qark/report.py

```diff
@@ -0,0 +1,57 @@
+from __future__ import absolute_import
+
+from os import path
+
+from jinja2 import Environment, PackageLoader, select_autoescape, Template
+
+from qark.vulnerability import (Vulnerability, Severity)
+
+DEFAULT_REPORT_PATH = '{}/report'.format(path.abspath(path.dirname(path.realpath(__file__)) + '/..'))
+
+jinja_env = Environment(
+    loader=PackageLoader('qark', 'templates'),
+    autoescape=select_autoescape(['html', 'xml'])
+)
+
+
+class Report(object):
+    """An object to store issues against and to generate reports in different formats.
+
+    There is one instance created per QARK run and it uses a classic Singleton pattern
+    to make it easy to get a reference to that instance anywhere in QARK.
+    """
+
+    # The one instance to rule them all
+    # http://python-3-patterns-idioms-test.readthedocs.io/en/latest/Singleton.html#the-singleton
+    __instance = None
+
+    def __new__(cls):
```

Comment:
Shouldn't this take a report path similar to the init?

Reply:
It does now! Nice catch.